packages/atomic/src/components/insight/atomic-insight-result/atomic-insight-result.pcss

```diff
@@ -5,3 +5,21 @@ atomic-result-section-children {
   border-left: 1px solid var(--atomic-neutral);
   padding-left: 1rem;
 }
+
+:host(.resultHovered) {
+  @apply relative;
+
+  .with-sections {
+    background-color: #f2f2f2;
```

Comment:
Colors should come from variables so if there isn't an existing one in `coveo.css` that works we can add one.